ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-nat.j2

```diff
@@ -0,0 +1,108 @@
+{# doc - ip nat #}
+{% if ip_nat is arista.avd.defined %}
+
+## IP NAT
+{%     if ip_nat.kernel_buffer_size is arista.avd.defined %}
+
+| Settings | Value |
+| -------- | ----- |
+| Kernel Buffer Size | {{ ip_nat.kernel_buffer_size }} MB |
+{%     endif %}
+{%     if ip_nat.pools is arista.avd.defined %}
+
+### NAT Pools
+{%         for pool in ip_nat.pools %}
+
+#### Pool: {{ pool.name }}
+
+| Settings | Value |
+| -------- | ----- |
+| Pool Prefix Length | {{ pool.prefix_length }} |
+{%             if pool.utilization_log_threshold is arista.avd.defined %}
+| Pool Utilization Threshold | {{ pool.utilization_log_threshold }} % (action: log) |
+{%             endif %}
+
+##### Pool Ranges
+
+| First IP Address  | Last IP Address | First Port | Last Port |
+| ----------------- | --------------- | ---------- | --------- |
+{%             for range in pool.ranges | arista.avd.default([]) %}
+| {{ range.first_ip }} | {{ range.last_ip }} | {{ range.first_port | arista.avd.default('n.a.') }} | {{ range.last_port | arista.avd.default('n.a.') }} |
```

Comment:
We tend to use `"-"` for unset values rather than n.a.

```suggestion
| {{ range.first_ip }} | {{ range.last_ip }} | {{ range.first_port | arista.avd.default("-") }} | {{ range.last_port | arista.avd.default("-") }} |
```

Reply:
Copy.

Reply:
Fixed in [ece7ab2](https://github.com/aristanetworks/ansible-avd/pull/2747/commits/ece7ab239d6f3915e0b9829b036a53e774724fce)